ui/raidboss/timeline.ts

```diff
@@ -70,10 +71,11 @@ const activeText = {
   ko: '시전중:',
 };
 
-type Replacement = {
+export type Replacement = {
   locale: string;
-  replaceSync: { [key: string]: string };
-  replaceText: { [key: string]: string };
+  missingTranslations?: boolean;
+  replaceSync?: { [key: string]: string };
+  replaceText?: { [key: string]: string };
```

Comment:
```suggestion
  replaceSync?: { [regexString: string]: string };
  replaceText?: { [timelineText: string]: string };
```

Just as slightly more informative names.